scout/ext/backend/utils/get_case.py

```diff
@@ -112,26 +114,32 @@ def get_case(scout_configs, family_type):
     logger.debug("Setting genome build to: {0}".format(
       scout_configs.get('human_genome_build', '')))
     
+    # Get the genome version
     mongo_case['genome_version'] = float(scout_configs.get('human_genome_version', '0'))
     logger.debug("Setting genome version to: {0}".format(
       scout_configs.get('human_genome_version', '0')))
 
+    # Check the analysis date
     mongo_case['analysis_date'] = scout_configs.get('analysis_date', '')
     logger.debug("Setting analysis date to: {0}".format(
       scout_configs.get('analysis_date', '')))
 
-    # Add the pedigree picture
+    # Add the pedigree picture, this is a xml file that will be read and 
+    # saved in the mongo database
     madeline_file = path(scout_configs.get('madeline', '/__menoexist.tXt'))
     if madeline_file.exists():
       logger.debug("Found madeline info")
       with madeline_file.open('r') as handle:
         mongo_case['madeline_info'] = handle.read()
+        logger.debug("Madeline file was read succesfully")
 
     # Add the coverage report
     coverage_report = scout_configs.get('coverage_report', None)
     if coverage_report:
-      mongo_case['coverage_report_path'] = coverage_report
-      logger.debug("Setting coverage report to: {0}".format(coverage_report))
+      logger.debug("Found a coverage report")
+      with coverage_report_path.open('rb') as handle:
+        mongo_case['coverage_report_path'] = handle.read()
```

Comment:
Shouldn't we rename this field to something more relevant now that it isn't a path?